test_runner/performance/pageserver/util.py

```diff
@@ -52,4 +52,12 @@ def doit(neon_env_builder: NeonEnvBuilder) -> NeonEnv:
     env = neon_env_builder.build_and_use_snapshot(name, doit)
     env.start()
     ensure_pageserver_ready_for_benchmarking(env, n_tenants)
+
+    env.pageserver.allowed_errors.append(
+        # https://github.com/neondatabase/neon/issues/6925
+        # https://github.com/neondatabase/neon/issues/6390
+        # https://github.com/neondatabase/neon/issues/6724
+        r".*query handler for.*pagestream.*failed: unexpected message: CopyFail during COPY.*"
+    )
```

Comment:
If this error happens only in `test_pageserver_max_throughput_getpage_at_latest_lsn` (that we unskip in the PR), I would add this allowed error only to the test, instead of `setup_pageserver_with_tenants` which we use in several other tests also.

Reply:
I had checked usage of `setup_pageserver_with_tenants`  - it is only used in two pageserver test cases - one that gets base backup and one that gets pages.
But I agree it is safer to only change the one testcase that has the problem